src/PlayerOptions.cpp

```diff
@@ -2509,12 +2597,15 @@ class LunaPlayerOptions: public Luna<PlayerOptions>
 		ADD_MULTICOL_METHOD(Bumpy);
 		ADD_MULTICOL_METHOD(Reverse);
 
-
 		ADD_METHOD(NoteSkin);
 		ADD_METHOD(FailSetting);
 		ADD_METHOD(MinTNSToHideNotes);
 		ADD_METHOD(VisualDelay);
 
+		ADD_METHOD(DisableTimingWindow);
+		ADD_METHOD(ResetDisabledTimingWindows);
+		ADD_METHOD(GetDisabledTimingWindows);
```

Comment:
Please also add to the lua docs.

Reply:
Done! I'm not sure how to re-render the page though.